specification/ml/evaluate_data_frame/types.ts

```diff
@@ -22,24 +22,65 @@ import { Name } from '@_types/common'
 import { double, integer } from '@_types/Numeric'
 
 export class DataframeOutlierDetectionSummary {
+  /**
+   * The AUC ROC (area under the curve of the receiver operating characteristic) score and optionally the curve.
+   * @server_default "include_curve": false
```

Comment:
I notice that the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/master/evaluate-dfanalytics.html#oldetection-resources) surround the default value by curly brackets. Should this perhaps be `{"include_curve": false}`?

Reply:
Thank you for the review! I updated the PR via [68e51f0](https://github.com/elastic/elasticsearch-specification/pull/2245/commits/68e51f00dd8fe63db4a158cd557979d437ae6e57).